js-exercises/is-triangle/isTriangle.js

```diff
@@ -0,0 +1,18 @@
+
+function isTriangle (a, b, c) {
+  if ((a + b <= c) || (a + c <= b) || (b + c <= a)) {
```

Comment:
These conditions itself return true/false. So, instead of if then else you can directly return them.